js/parts-gantt/Pathfinder.js

```diff
@@ -1249,10 +1248,26 @@ extend(H.Point.prototype, /** @lends Point.prototype */ {
     }
 });
 
+
+// Warn if using legacy options
+function warnLegacy(chart) {
+    if (
+        chart.options.pathfinder ||
+        H.reduce(chart.series, function (acc, series) {
+            return acc || series.options && series.options.pathfinder;
+        }, false)
+    ) {
+        H.error('Pathfinder options have been renamed. ' +
```

Comment:
I find it a bit unfortunate that things will break for existing users, before they had a chance to fix the configuration. Could we consider temporarily copying `series.options.pathfinder` to `series.options.connectors`, and the same for `chart.options`?
Then in the warning we could say that it is temporarily and will be removed in one of the upcoming versions.